scripts/ansible/aws/ec2-set-volume-size.yml

```diff
@@ -19,7 +19,8 @@
 
   - name: Modify the volume
```

Comment:
A better experience is for this script to expect the size by which to increase the disk and not the final size; saves a trip to the AWS console. I don't expect this PR to implement this.

Reply:
agreed